prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="file:///C:/Users/kirei/repos/prep-0721-code-solutions/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
Rather than use the absolute file path to the `index.html`, use a relative path to the `index.html` in the parent directory (_i.e._ `href="../index.html"`). 

That way, the link works on computers other than your own.